server.rb

```diff
@@ -74,14 +74,17 @@ def comment_created_message(redis, repository, issue, comment)
   end
 end
 
-def pull_request_opened_message(redis, repository, pull_request)
-  score = redis.get(pull_request.user.login).to_i
+def handle_labeled_event(event, redis, slack)
+  repository = event.repository
+  issue = event.issue
+  return unless event.labeled?
```

Comment:
move this out into the `post` block so that it follows the same pattern as the `comment_created?` flow.